dashboard/app/views/levels/_apps_dependencies.html.haml

```diff
@@ -29,6 +29,9 @@
   %script{src: "#{static_asset_base_path}js/ace/ext-language_tools.js"}
   %script{src: asset_path('js/droplet/droplet-full.js')}
   %script{src: asset_path('js/tooltipster/jquery.tooltipster.js')}
+  %script{src: asset_path('js/fileupload/jquery.iframe-transport.js')}
+  %script{src: asset_path('js/fileupload/jquery.fileupload.js')}
+
```

Comment:
minimized jquery.iframe-transport.js is only 2.2k, so probably not worth optimizing for